library/modules/bigip_profile_client_ssl.py

```diff
@@ -173,6 +173,12 @@
       - once
       - always
     version_added: 2.8
+  renegotiation:
+    description:
+      - Enables or disables SSL renegotiation.  The default value is enabled.
```

Comment:
no need to specify the default value here just use default: yes, if this is referring to system default variable there is no need to provide this information. 

Just mention that this value is inherited from parent profile upon creation